ui-lib/src/main/res/ui/warning/values/strings.xml

```diff
@@ -4,4 +4,11 @@
     <string name="not_enough_space_description">You need approximately <xliff:g example="1" id="required_gigabytes">%1$s</xliff:g> gig of space while synchronizing the Zcash blockchain, but only 300 megs once done. Syncing will stay paused until more space is available.</string>
     <string name="space_required_to_continue"><xliff:g example="300" id="required_megabytes">~%1$s</xliff:g> megs required to continue </string>
     <string name="unknown">Unknown</string>
+    <string name="security_warning_text">Zashi 0.9 is a Zcash-only shielded wallet, built by Zcashers for Zcashers. The purpose of this release is primarily to test functionality and collect feedback. While Zashi has been engineered for your privacy and safety, this release has not yet been security audited. Users are cautioned to deposit, send, and receive only small amounts of ZEC. Please click below to proceed.</string>
```

Comment:
Could we make the Zashi version text dynamic with xliff so it's not hardcoded into the string itself?